pmlc/conversion/stdx_to_llvm/stdx_to_llvm.cc

```diff
@@ -345,6 +345,42 @@ struct CmpXchgLowering : public LoadStoreOpLowering<stdx::AtomicRMWOp> {
   }
 };
 
+struct FPToSILowering : public LLVMLegalizationPattern<stdx::FPToSIOp> {
+  using LLVMLegalizationPattern<stdx::FPToSIOp>::LLVMLegalizationPattern;
+  using Base = LLVMLegalizationPattern<stdx::FPToSIOp>;
+
+  PatternMatchResult
+  matchAndRewrite(Operation *op, ArrayRef<Value> operands,
+                  ConversionPatternRewriter &rewriter) const override {
+    if (auto fptosiOp = cast<stdx::FPToSIOp>(op)) {
+      auto value = fptosiOp.getOperand();
+      auto stdxType = fptosiOp.getOperation()->getResult(0).getType();
+      auto llvmType = typeConverter.convertType(stdxType);
```

Comment:
Actually, we don't need to do this, this should already be done via the `operands` being passed to `matchAndRewrite`.